<!doctype html><html lang=zh-tw><head><meta charset=utf-8><meta name=description content="[!info]
🌱 來自：[[researcher]]
 bioinfo [[ngs.md|NGS]] [[ngs_tools.md|NGS tools]]
[[genomics.md|Genomics]] [[bioinformatics_algorithms.md|bioinformatics algorithms]]
Chapter 1 Sequencing and Raw Sequence Data Quality Control   1."><meta property="og:title" content><meta property="og:description" content><meta property="og:type" content="website"><meta property="og:url" content><meta property="og:image" content="https://og-mlkgi530p-htlin222.vercel.app/api/blog?templateTitle=bioinfo"><title>bioinfo</title><meta name=viewport content="width=device-width,initial-scale=1"><link rel=alternate type=application/rss+xml href=https://www.physician.tw//feed.xml title><link rel="shortcut icon" type=image/png href=https://www.physician.tw//icon.png><link href=https://www.physician.tw/styles.acd6336f837a791cf1889bb120dfb142.min.css rel=stylesheet><link href=https://www.physician.tw/styles/_light_syntax.86a48a52faebeaaf42158b72922b1c90.min.css rel=stylesheet id=theme-link><script src=https://www.physician.tw/js/darkmode.dfadb738e747a87e78c7144d7267f1bc.min.js></script>
<script src=https://www.physician.tw/js/util.6f22941e242efae60fd84e7c32e874fa.min.js></script>
<link rel=preload href=https://cdn.jsdelivr.net/npm/katex@0.15.1/dist/katex.min.css as=style onload='this.onload=null,this.rel="stylesheet"' integrity=sha384-R4558gYOUz8mP9YWpZJjofhk+zx0AS11p36HnD2ZKj/6JR5z27gSSULCNHIRReVs crossorigin=anonymous><script defer src=https://cdn.jsdelivr.net/npm/katex@0.15.1/dist/katex.min.js integrity=sha384-z1fJDqw8ZApjGO3/unPWUPsIymfsJmyrDVWC8Tv/a1HeOtGmkwNd/7xUS0Xcnvsx crossorigin=anonymous></script>
<script defer src=https://cdn.jsdelivr.net/npm/katex@0.15.1/dist/contrib/auto-render.min.js integrity=sha384-+XBljXPPiv+OzfbB3cVmLHf4hdUFHlWNZN5spNQ7rmHTXpd7WvJum6fIACpNNfIR crossorigin=anonymous></script>
<script defer src=https://cdn.jsdelivr.net/npm/katex@0.16.2/dist/contrib/copy-tex.min.js integrity=sha384-ww/583aHhxWkz5DEVn6OKtNiIaLi2iBRNZXfJRiY1Ai7tnJ9UXpEsyvOITVpTl4A crossorigin=anonymous></script>
<script async src=https://unpkg.com/@floating-ui/core@0.7.3></script>
<script async src=https://unpkg.com/@floating-ui/dom@0.5.4></script>
<script async src=https://www.physician.tw/js/popover.f03552ccb84d99ca615d1cfb9abde59e.min.js></script>
<script defer src=https://www.physician.tw/js/code-title.ce4a43f09239a9efb48fee342e8ef2df.min.js></script>
<script defer src=https://www.physician.tw/js/clipboard.2913da76d3cb21c5deaa4bae7da38c9f.min.js></script>
<script defer src=https://www.physician.tw/js/callouts.7723cac461d613d118ee8bb8216b9838.min.js></script>
<script>const SEARCH_ENABLED=!1,LATEX_ENABLED=!0,PRODUCTION=!0,BASE_URL="https://www.physician.tw/",fetchData=Promise.all([fetch("https://www.physician.tw/indices/linkIndex.f3f893e01d4e2f383f67a78640843e76.min.json").then(e=>e.json()).then(e=>({index:e.index,links:e.links})),fetch("https://www.physician.tw/indices/contentIndex.6e501fe1f761b18e889ba8bddd3bf89c.min.json").then(e=>e.json())]).then(([{index:e,links:t},n])=>({index:e,links:t,content:n})),render=()=>{const e=new URL(BASE_URL),t=e.pathname,n=window.location.pathname,s=t==n;addCopyButtons(),addTitleToCodeBlocks(),addCollapsibleCallouts(),initPopover("https://www.physician.tw",!0);const o=document.getElementById("footer");if(o){const e=document.getElementById("graph-container");if(!e)return requestAnimationFrame(render);e.textContent="";const t=s&&!1;drawGraph("https://www.physician.tw",t,[{"/moc":"#4388cc"}],t?{centerForce:1,depth:-1,enableDrag:!0,enableLegend:!1,enableZoom:!0,fontSize:.5,linkDistance:1,opacityScale:3,repelForce:1,scale:1.4}:{centerForce:1,depth:1,enableDrag:!0,enableLegend:!1,enableZoom:!0,fontSize:.6,linkDistance:1,opacityScale:3,repelForce:2,scale:1.2})}},init=(e=document)=>{addCopyButtons(),addTitleToCodeBlocks(),renderMathInElement(e.body,{delimiters:[{left:"$$",right:"$$",display:!0},{left:"$",right:"$",display:!1}],macros:{'’':"'"},throwOnError:!1})}</script><script type=module>
    import { attachSPARouting } from "https:\/\/www.physician.tw\/js\/router.9d4974281069e9ebb189f642ae1e3ca2.min.js"
    attachSPARouting(init, render)
  </script></head><body><div id=search-container><div id=search-space><input autocomplete=off id=search-bar name=search type=text aria-label=search placeholder="請輸入像是：hypertension 之類的來看沒有沒什麼有趣的🎳"><div id=results-container></div></div></div><script src=https://cdn.jsdelivr.net/npm/flexsearch@0.7.21/dist/flexsearch.bundle.js integrity="sha256-i3A0NZGkhsKjVMzFxv3ksk0DZh3aXqu0l49Bbh0MdjE=" crossorigin=anonymous defer></script>
<script defer src=https://www.physician.tw/js/full-text-search.e6e2e0c213187ca0c703d6e2c7a77fcd.min.js></script><div class=singlePage><header><h1 id=page-title><a href=https://www.physician.tw/>蜥蜴花園🪴</a></h1><div class=spacer></div><div id=search-icon><p>search</p><svg tabindex="0" aria-labelledby="title desc" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 19.9 19.7"><title id="title">搜尋圖示</title><desc id="desc">打開搜尋圖標</desc><g class="search-path" fill="none"><path stroke-linecap="square" d="M18.5 18.3l-5.4-5.4"/><circle cx="8" cy="8" r="7"/></g></svg></div><div class=darkmode><input class=toggle id=darkmode-toggle type=checkbox tabindex=-1>
<label id=toggle-label-light for=darkmode-toggle tabindex=-1><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" id="dayIcon" viewBox="0 0 35 35" style="enable-background:new 0 0 35 35"><title>明亮模式</title><path d="M6 17.5C6 16.672 5.328 16 4.5 16h-3C.672 16 0 16.672.0 17.5S.672 19 1.5 19h3C5.328 19 6 18.328 6 17.5zM7.5 26c-.414.0-.789.168-1.061.439l-2 2C4.168 28.711 4 29.086 4 29.5 4 30.328 4.671 31 5.5 31c.414.0.789-.168 1.06-.44l2-2C8.832 28.289 9 27.914 9 27.5 9 26.672 8.329 26 7.5 26zm10-20C18.329 6 19 5.328 19 4.5v-3C19 .672 18.329.0 17.5.0S16 .672 16 1.5v3C16 5.328 16.671 6 17.5 6zm10 3c.414.0.789-.168 1.06-.439l2-2C30.832 6.289 31 5.914 31 5.5 31 4.672 30.329 4 29.5 4c-.414.0-.789.168-1.061.44l-2 2C26.168 6.711 26 7.086 26 7.5 26 8.328 26.671 9 27.5 9zM6.439 8.561C6.711 8.832 7.086 9 7.5 9 8.328 9 9 8.328 9 7.5c0-.414-.168-.789-.439-1.061l-2-2C6.289 4.168 5.914 4 5.5 4 4.672 4 4 4.672 4 5.5c0 .414.168.789.439 1.06l2 2.001zM33.5 16h-3c-.828.0-1.5.672-1.5 1.5s.672 1.5 1.5 1.5h3c.828.0 1.5-.672 1.5-1.5S34.328 16 33.5 16zM28.561 26.439C28.289 26.168 27.914 26 27.5 26c-.828.0-1.5.672-1.5 1.5.0.414.168.789.439 1.06l2 2C28.711 30.832 29.086 31 29.5 31c.828.0 1.5-.672 1.5-1.5.0-.414-.168-.789-.439-1.061l-2-2zM17.5 29c-.829.0-1.5.672-1.5 1.5v3c0 .828.671 1.5 1.5 1.5s1.5-.672 1.5-1.5v-3C19 29.672 18.329 29 17.5 29zm0-22C11.71 7 7 11.71 7 17.5S11.71 28 17.5 28 28 23.29 28 17.5 23.29 7 17.5 7zm0 18c-4.136.0-7.5-3.364-7.5-7.5s3.364-7.5 7.5-7.5 7.5 3.364 7.5 7.5S21.636 25 17.5 25z"/></svg></label><label id=toggle-label-dark for=darkmode-toggle tabindex=-1><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" id="nightIcon" viewBox="0 0 100 100" style="enable-background='new 0 0 100 100'"><title>黑暗模式</title><path d="M96.76 66.458c-.853-.852-2.15-1.064-3.23-.534-6.063 2.991-12.858 4.571-19.655 4.571C62.022 70.495 50.88 65.88 42.5 57.5 29.043 44.043 25.658 23.536 34.076 6.47c.532-1.08.318-2.379-.534-3.23-.851-.852-2.15-1.064-3.23-.534-4.918 2.427-9.375 5.619-13.246 9.491-9.447 9.447-14.65 22.008-14.65 35.369.0 13.36 5.203 25.921 14.65 35.368s22.008 14.65 35.368 14.65c13.361.0 25.921-5.203 35.369-14.65 3.872-3.871 7.064-8.328 9.491-13.246C97.826 68.608 97.611 67.309 96.76 66.458z"/></svg></label></div><link rel=alternate type=application/rss+xml href=https://www.physician.tw/bioinfo/rss.xml title></header><article><h1>bioinfo</h1><p class=meta>最後更新於
Apr 15, 2024
<a href=https://github.com/htlin222/garden/tree/hugo/content/bioinfo.md rel=noopener>🪚 編輯原始文件</a></p><ul class=tags><li><a href=https://www.physician.tw/tags/building/>building</a></li></ul><blockquote class=info-callout><p></p><p>🌱 來自：<a href=/researcher rel=noopener class=internal-link data-src=/researcher>researcher</a></p></blockquote><a href=#bioinfo><h1 id=bioinfo><span class=hanchor arialabel=Anchor># </span>bioinfo</h1></a><p><a class="internal-link broken">NGS</a>
<a href=/ngstools rel=noopener class=internal-link data-src=/ngstools>NGS tools</a></p><p><a href=/genomics rel=noopener class=internal-link data-src=/genomics>Genomics</a>
<a href=/bioinformaticsalgorithms rel=noopener class=internal-link data-src=/bioinformaticsalgorithms>bioinformatics algorithms</a></p><a href=#chapter-1-sequencing-and-raw-sequence-data-quality-control><h2 id=chapter-1-sequencing-and-raw-sequence-data-quality-control><span class=hanchor arialabel=Anchor># </span>Chapter 1 Sequencing and Raw Sequence Data Quality Control</h2></a><ul><li><p>1.1 NUCLEIC ACIDS 1</p></li><li><p>1.2 SEQUENCING 3</p></li><li><p>1.2.1 <a href=/first-generationsequencing rel=noopener class=internal-link data-src=/first-generationsequencing>First-Generation Sequencing</a> 3</p></li><li><p>1.2.2 Next-Generation Sequencing 4</p></li><li><p>1.2.2.1 Roche 454 Technology 5</p></li><li><p>1.2.2.2 Ion Torrent Technology 6</p></li><li><p>1.2.2.3 AB SOLiD Technology 6</p></li><li><p>1.2.2.4 Illumina Technology 7</p></li><li><p>1.2.3 Third-Generation Sequencing 8</p></li><li><p>1.2.3.1 PacBio Technology 9</p></li><li><p>1.2.3.2 Oxford Nanopore Technology 10</p></li><li><p>1.3 SEQUENCING DEPTH AND READ QUALITY 11</p></li><li><p>1.3.1 Sequencing Depth 11</p></li><li><p>1.3.2 Base Call Quality 11</p></li><li><p>1.4 <a href=/fastqfiles rel=noopener class=internal-link data-src=/fastqfiles>FASTQ FILES</a> 13</p></li><li><p>1.5 FASTQ READ QUALITY ASSESSMENT 18</p></li><li><p>1.5.1 Basic Statistics 23</p></li><li><p>1.5.2 Per Base Sequence Quality 24</p></li><li><p>1.5.3 Per Tile Sequence Quality 25</p></li><li><p>1.5.4 Per Sequence Quality Scores 28</p></li><li><p>1.5.5 Per Base Sequence Content 28</p></li><li><p>1.5.6 Per Sequence GC Content 28</p></li><li><p>1.5.7 Per Base N Content 30</p></li><li><p>1.5.8 Sequence Length Distribution 30</p></li><li><p>1.5.9 Sequence Duplication Levels 31</p></li><li><p>1.5.10 Overrepresented Sequences 31</p><ul><li><a href=/trimmingwithtrimmomatic rel=noopener class=internal-link data-src=/trimmingwithtrimmomatic>trimming with trimmomatic</a></li></ul></li><li><p>1.5.11 Adapter Content 32</p></li><li><p>1.5.12 K-mer Content 33</p></li><li><p>1.6 PREPROCESSING OF THE FASTQ READS 34</p></li><li><p>1.7 SUMMARY 45</p></li><li><p>REFERENCES 46</p></li><li><p>Chapter 2 ◾ Mapping of Sequence Reads to the Reference Genomes 49</p></li><li><p>2.1 INTRODUCTION TO SEQUENCE MAPPING 49</p></li><li><p>2.2 READ MAPPING 55</p></li><li><p>2.2.1 Trie 56</p></li><li><p>2.2.2 Suffix Tree 56</p></li><li><p>2.2.3 Suffix Arrays 57</p></li><li><p>2.2.4 Burrows-Wheeler Transform 58</p></li><li><p>2.2.5 FM-Index 62</p></li><li><p>2.3 READ SEQUENCE ALIGNMENT AND ALIGNERS 63</p></li><li><p>2.3.1 SAM and BAM File Formats 65</p></li><li><p>2.3.2 Read Aligners 70</p></li><li><p>2.3.2.1 Burrows-Wheeler Aligner 71</p></li><li><p>2.3.2.2 Bowtie2 75</p></li><li><p>2.3.2.3 STAR 76</p></li><li><p>2.4 MANIPULATING ALIGNMENTS IN SAM/BAM FILES 79</p></li><li><p>2.4.1 <a href=/samtools rel=noopener class=internal-link data-src=/samtools>samtools</a> 79</p></li><li><p>2.4.1.1 SAM/BAM Format Conversion 79</p></li><li><p>2.4.1.2 Sorting Alignment 80</p></li><li><p>2.4.1.3 Indexing BAM File 80</p></li><li><p>2.4.1.4 Extracting Alignments of a Chromosome 81</p></li><li><p>2.4.1.5 Filtering and Counting Alignment in SAM/BAM Files 81</p></li><li><p>2.4.1.6 Removing Duplicate Reads 82</p></li><li><p>2.4.1.7 Descriptive Statistics 83</p></li><li><p>2.5 REFERENCE-GUIDED GENOME ASSEMBLY 83</p></li><li><p>Chapter 3 ◾ De Novo Genome Assembly 89</p></li><li><p>3.1 INTRODUCTION TO DE NOVO GENOME ASSEMBLY 89</p></li><li><p>3.1.1 Greedy Algorithm 90</p></li><li><p>3.1.2 Overlap-Consensus Graphs 90</p></li><li><p>3.1.3 De Bruijn Graphs 91</p></li><li><p>3.2 EXAMPLES OF DE NOVO ASSEMBLERS 93 3.2.1 ABySS 93 3.2.2 SPAdes 97</p></li><li><p>3.3 GENOME ASSEMBLY QUALITY ASSESSMENT 99</p></li><li><p>3.3.1 Statistical Assessment for Genome Assembly 100</p></li><li><p>3.3.2 Evolutionary Assessment for De Novo Genome Assembly 103</p></li><li><p>3.4 SUMMARY 106</p></li><li><p>Chapter 4 ◾ <a href=/variantdiscovery rel=noopener class=internal-link data-src=/variantdiscovery>variant discovery</a> 109</p></li><li><p>4.1 INTRODUCTION TO GENETIC VARIATIONS 109</p></li><li><p>4.1.1 VCF File Format 110</p></li><li><p>4.1.2. Variant Calling and Analysis 113</p></li><li><p>4.2 VARIANT CALLING PROGRAMS 114</p></li><li><p>4.2.1 <a href=/consensus-basedvariantcallers rel=noopener class=internal-link data-src=/consensus-basedvariantcallers>Consensus-Based Variant Callers</a> 114</p></li><li><p>4.2.1.1 BCF Tools Variant Calling Pipeline 115</p></li><li><p>4.2.2 Haplotype-Based Variant Callers 125</p></li><li><p>4.2.2.1 FreeBayes Variant Calling Pipeline 127</p></li><li><p>4.2.2.2 GATK Variant Calling Pipeline 129</p></li><li><p>4.3 VISUALIZING VARIANTS 143</p></li><li><p>4.4 VARIANT ANNOTATION AND PRIORITIZATION 143</p></li><li><p>4.4.1 SIFT 145</p></li><li><p>4.4.2 SnpEff 148</p></li><li><p>4.3.3 ANNOVAR 151</p></li><li><p>4.3.3.1</p></li><li><p>4.3.3.2 ANNOVAR Input Files 156 160 161</p></li><li><p>4.5 SUMMARY REFERENCES</p></li><li><p>Annotation Databases 153</p></li><li><p>Chapter 5 ◾ RNA-Seq Data Analysis 163</p></li><li><p>5.1 INTRODUCTION TO RNA-SEQ 163</p></li><li><p>5.2 RNA-SEQ APPLICATIONS 165</p></li><li><p>5.3 RNA-SEQ DATA ANALYSIS WORKFLOW 166</p></li><li><p>5.3.1 Acquiring RNA-Seq Data 166</p></li><li><p>5.3.2 Read Mapping 167</p></li><li><p>5.3.3 Alignment Quality Assessment 171</p></li><li><p>5.3.4 Quantification 172</p></li><li><p>5.3.5 Normalization 174</p></li><li><p>5.3.5.1 RPKM and FPKM 174</p></li><li><p>5.3.5.2 Transcripts per Million 175</p></li><li><p>5.3.5.3 Counts per Million Mapped Reads 175</p></li><li><p>5.3.5.4 Trimmed Mean of M-values 175</p></li><li><p>5.3.5.5 Relative Expression 176</p></li><li><p>5.3.5.6 Upper Quartile 176</p></li><li><p>5.3.6 Differential Expression Analysis 176</p></li><li><p>5.3.7 Using EdgeR for Differential Analysis 180</p></li><li><p>5.3.7.1 Data Preparation 181</p></li><li><p>5.3.7.2 Annotation 183</p></li><li><p>5.3.7.3 Design Matrix 184</p></li><li><p>5.3.7.4 Filtering Low-Expressed Genes 185</p></li><li><p>5.3.7.5 Normalization 186</p></li><li><p>5.3.7.6 Estimating Dispersions 186</p></li><li><p>5.3.7.7 Exploring the Data 189</p></li><li><p>5.3.7.8 Model Fitting 194</p></li><li><p>5.3.7.9 Ontology and Pathways 202</p></li><li><p>5.3.8 Visualizing RNA-Seq Data 204</p></li><li><p>5.3.8.1 5.3.8.2 5.3.8.3 5.3.8.4</p></li><li><p>Visualizing Distribution with Boxplots 206 Scatter Plot 207 Mean-Average Plot (MA Plot) 208 Volcano Plots 209</p></li><li><p>209 211</p></li><li><p>Chapter 6 ◾ Chromatin Immunoprecipitation Sequencing 213</p></li><li><p>6.1 INTRODUCTION TO CHROMATIN IMMUNOPRECIPITATION 213</p></li><li><p>6.2 CHIP SEQUENCING 214</p></li><li><p>6.3 CHIP-SEQ ANALYSIS WORKFLOW 215</p></li><li><p>6.3.1 Downloading the Raw Data 217</p></li><li><p>6.3.2 Quality Control 218</p></li><li><p>6.3.3 ChIP-Seq and Input Read Mapping 219</p></li><li><p>6.3.4 ChIP-Seq Peak Calling with MACS3 223</p></li><li><p>6.3.5 Visualizing ChIP-Seq Enrichment in Genome Browser 226</p></li><li><p>6.3.6 Visualizing Peaks Distribution 229</p></li><li><p>6.3.6.1 ChIP-Seq Peaks’ Coverage Plot 230</p></li><li><p>6.3.6.2 Distribution of Peaks in Transcription Start Site (TSS)</p></li><li><p>Regions 233</p></li><li><p>6.3.6.3 Profile of Peaks along Gene Regions 234</p></li><li><p>6.3.7 Peak Annotation 235</p></li><li><p>6.3.7.1 Writing Annotations to Files 237</p></li><li><p>6.3.8 ChIP-Seq Functional Analysis 239</p></li><li><p>6.3.9 Motif Discovery 243</p></li><li><p>6.4 SUMMARY 250 REFERENCES 251</p></li><li><p>Chapter 7 ◾ Targeted Gene Metagenomic Data Analysis 253 7.1. INTRODUCTION TO METAGENOMICS 253 7.2 ANALYSIS WORKFLOW 254</p></li><li><p>7.2.1 Raw Data Preprocessing 254</p></li><li><p>7.2.2 Metagenomic Features 255 7.2.2.1 Clustering 255 7.2.2.2 Denoising 256</p></li><li><p>7.2.3 Taxonomy Assignment 258</p></li><li><p>7.2.3.1 Basic Local Alignment Search Tool 258</p></li><li><p>7.2.3.2 VSEARCH 259</p></li><li><p>7.2.3.3 Ribosomal Database Project 259</p></li><li><p>7.2.4 Construction of Phylogenetic Trees 260</p></li><li><p>7.2.5 Microbial Diversity Analysis 261</p></li><li><p>7.2.5.1 Alpha Diversity Indices 262</p></li><li><p>7.2.5.2 Beta Diversity 262</p></li><li><p>7.3 DATA ANALYSIS WITH QIIME2 263</p></li><li><p>7.3.1 QIIME2 Input Files 265 7.3.1.1 Importing Sequence Data 265 7.3.1.2 Metadata 269</p></li><li><p>7.3.2 Demultiplexing 269</p></li><li><p>7.3.3 Downloading and Preparing the Example Data 271</p></li><li><p>7.3.3.1 Downloading the Raw Data 271</p></li><li><p>7.3.3.2 Creating the Sample Metadata File 272</p></li><li><p>7.3.3.3 Importing Microbiome Yoga Data 274</p></li><li><p>7.3.4 Raw Data Preprocessing 275</p></li><li><p>7.3.4.1 Quality Assessment and Quality Control 275</p></li><li><p>7.3.4.2 Clustering and Denoising 278</p></li><li><p>7.3.5 Taxonomic Assignment with QIIME2 289</p></li><li><p>7.3.5.1 Using Alignment-Based Classifiers 289</p></li><li><p>7.3.5.2 Using Machine Learning Classifiers 291</p></li><li><p>7.3.6 Construction of Phylogenetic Tree 297</p></li><li><p>7.3.6.1 De Novo Phylogenetic Tree 297</p></li><li><p>7.3.6.2 Fragment-Insertion Phylogenetic Tree 298</p></li><li><p>7.3.7 Alpha and Beta Diversity Analysis 298</p></li><li><p>7.4 SUMMARY 300 REFERENCES 301</p></li><li><a href=#chapter-8-shotgun-metagenomic-data-analysis-303><h2 id=chapter-8-shotgun-metagenomic-data-analysis-303><span class=hanchor arialabel=Anchor># </span>Chapter 8 Shotgun Metagenomic Data Analysis 303</h2></a><p><a href=https://zhuanlan.zhihu.com/p/128653464 rel=noopener>初探 PLINK 檔案格式（bed，bim，fam） - 知乎</a>
<a href=https://zzz.bwh.harvard.edu/plink/ rel=noopener>PLINK: Whole genome data analysis toolset</a>
<a href=https://geneviatechnologies.com/blog/clonal-evolutionary-analysis-of-cancer/ rel=noopener>Clonal evolutionary analysis of cancer - Genevia Technologies</a></p></li><li><p>8.1 INTRODUCTION 303</p></li><li><p>8.2 SHOTGUN METAGENOMIC ANALYSIS WORKFLOW 305</p></li><li><p>8.2.1 Data Acquisition 305</p></li><li><p>8.2.2 Quality Assessment and Processing 305</p></li><li><p>8.2.3 Removing Host DNA Reads 306</p></li><li><p>8.2.3.1 Download Human Reference Genome 306</p></li><li><p>8.2.3.2 Mapping Reads to the Reference Genome 307</p></li><li><p>8.2.3.3 Converting SAM to BAM Format 307</p></li><li><p>8.2.3.4 Separating Metagenomic Reads in BAM Files 307</p></li><li><p>8.2.3.5 Creating Paired-End FASTQ Files from BAM Files 308</p></li><li><p>8.2.4 Assembly-Free Taxonomic Profiling 310</p></li><li><p>8.2.4 Assembly of Metagenomes 315</p></li><li><p>8.2.5 Assembly Evaluation 317</p></li><li><p>8.2.6 Mapping Reads to the Assemblies 318</p></li><li><p>8.2.7 Binning 321</p></li><li><p>8.2.8 Bin Evaluation 323</p></li><li><p>8.2.9 Prediction of Protein-Coding Region</p></li></ul><p><a href=/ngsarticles rel=noopener class=internal-link data-src=/ngsarticles>ngs_articles</a></p></article><hr><div class=page-end id=footer><div class=backlinks-container><h3>↩️ 反向連結</h3><ul class=backlinks><li><a href=/bioinformatics_algorithms/ data-ctx=bioinfo data-src=/bioinformatics_algorithms class=internal-link>bioinformatics_algorithms</a></li><li><a href=/consensus-based_variant_callers/ data-ctx=bioinfo data-src=/consensus-based_variant_callers class=internal-link>consensus-based_variant_callers</a></li><li><a href=/fastq_files/ data-ctx=bioinfo data-src=/fastq_files class=internal-link>fastq_files</a></li><li><a href=/first-generation_sequencing/ data-ctx=bioinfo data-src=/first-generation_sequencing class=internal-link>first-generation_sequencing</a></li><li><a href=/genomics/ data-ctx=bioinfo data-src=/genomics class=internal-link>genomics</a></li><li><a href=/ngs_articles/ data-ctx=bioinfo data-src=/ngs_articles class=internal-link>ngs_articles</a></li><li><a href=/ngs_tools/ data-ctx=bioinfo data-src=/ngs_tools class=internal-link>ngs_tools</a></li><li><a href=/oncology/ data-ctx=bioinfo data-src=/oncology class=internal-link>oncology</a></li><li><a href=/researcher/ data-ctx=bioinfo data-src=/researcher class=internal-link>researcher</a></li><li><a href=/samtools/ data-ctx=bioinfo data-src=/samtools class=internal-link>samtools</a></li><li><a href=/trimming_with_trimmomatic/ data-ctx=bioinfo data-src=/trimming_with_trimmomatic class=internal-link>trimming_with_trimmomatic</a></li><li><a href=/variant_discovery/ data-ctx=bioinfo data-src=/variant_discovery class=internal-link>variant_discovery</a></li></ul></div><div><script async src=https://cdn.jsdelivr.net/npm/d3@6.7.0/dist/d3.min.js integrity="sha256-+7jaYCp29O1JusNWHaYtgUn6EhuP0VaFuswhNV06MyI=" crossorigin=anonymous></script><h3>🤯互動圖</h3><div id=graph-container></div><style>:root{--g-node:var(--secondary);--g-node-active:var(--primary);--g-node-inactive:var(--visited);--g-link:var(--outlinegray);--g-link-active:#5a7282}</style><script src=https://www.physician.tw/js/graph.2d9e48dbe7ea47c0ef1c58296ce14448.js></script></div></div><div><div id=disqus_thread></div><script type=application/javascript>window.disqus_config=function(){},function(){if(["localhost","127.0.0.1"].indexOf(window.location.hostname)!=-1){document.getElementById("disqus_thread").innerHTML="Disqus comments not available by default when the website is previewed locally.";return}var t=document,e=t.createElement("script");e.async=!0,e.src="//physician.disqus.com/embed.js",e.setAttribute("data-timestamp",+new Date),(t.head||t.body).appendChild(e)}()</script><noscript>Please enable JavaScript to view the <a href=https://disqus.com/?ref_noscript>comments powered by Disqus.</a></noscript><a href=https://disqus.com class=dsq-brlink>comments powered by <span class=logo-disqus>Disqus</span></a></div><div id=contact_buttons><footer><p>本網站是由 Hsieh-Ting Lin 以❤️ 用 <a href=https://github.com/jackyzha0/quartz>Quartz</a>這個酷東西種出來🪴 , © 2024</p><p><a href=https://www.buymeacoffee.com/htl.lizard><img src=https://cdn.buymeacoffee.com/buttons/v2/default-yellow.png width=200></a></p><ul><li><a href=https://www.physician.tw/>🏠首頁</a></li><li><a href=https://www.facebook.com/htlin.lizard>Facebook</a></li><li><a href=https://twitter.com/htlin222>Twitter</a></li><li><a href=https://github.com/htlin222>GitHub</a></li><li><a href=https://htlin.site>Blog</a></li><li><a href=https://htlin222.medium.com>Medium</a></li></ul></footer></div></div></body></html>